en/docs/reference/mediators/switch-mediator.md

```diff
@@ -28,44 +28,40 @@ The parameters available to configure the Switch mediator are as follows.
 </thead>
 <tbody>
 <tr class="odd">
-<td><strong>Source XPath</strong></td>
-<td>The source XPath or JSONPath to be evaluated. When specifying a JSONPath, use the format <code>json-eval(&lt;JSON_PATH&gt;)</code> , such as <code>json-eval(getQuote.request.symbol)</code> . If you use namespaces in the expression, click the <strong>edit icon</strong>, then click <strong>Add Namespace</strong> to map the namespace prefix to the correct URI.
+<td><strong>Expression</strong></td>
+<td>The Synapse expression to be evaluated which should return a string.</td>
 </tr>
 <tr class="even">
-<td><strong>Number of cases</strong></td>
-<td><p>This parameter displays the number of cases currently added to the Switch mediator configuration.</p>
+<td><strong>Cases</strong></td>
+<td><p>This section displays the number of cases currently added to the switch mediator and option to add a new case.</p>
 <p><br />
 </p></td>
 </tr>
-<tr class="odd">
-<td><strong>Specify default case</strong></td>
-<td>Adding a default case is optional. If it is specified, it will be executed if no matching case is identified.</td>
-</tr>
 </tbody>
 </table>
 
-## Switch-case mediator
+### Adding a new case
 
-1.  To add a case, go to edit the switch and click **Add parameter** under **Case Branches**. 
-2.  It will open a form where a regular expression can be added in the **Case Regex** parameter. By submitting, it will add an empty case under the switch mediator as a child. 
-3.  Click on the `+` mark under a chase to add the mediator(s) you want to execute when this case matches the switching value.
+1.  To add a case, go to edit the switch mediator and click **Add new case** under **Cases** section.
+2.  It will open a form where a regular expression can be added in the **Regex Pattern** parameter. By submitting, it will add an empty case under the switch mediator as a child.
```

Comment:
```suggestion
2.  It will open a form where a regular expression can be added in the **Regex Pattern** parameter. Once submitted, it will add an empty case under the switch mediator as a child.
```